Jellyfin.Api/ModelBinders/CommaDelimitedArrayModelBinder.cs

```diff
@@ -55,5 +55,36 @@ public Task BindModelAsync(ModelBindingContext bindingContext)
 
             return Task.CompletedTask;
         }
+
+        private Array GetParsedResult(IReadOnlyList<string> values, Type elementType, TypeConverter converter)
+        {
+            var parsedValues = new object?[values.Count];
+            var convertedCount = 0;
+            for (var i = 0; i < values.Count; i++)
+            {
+                try
+                {
+                    parsedValues[i] = converter.ConvertFromString(values[i]?.Trim());
```

Comment:
If values is an ```IReadOnlyList<string>``` shouldn't values[i] always have a value? (otherwise wouldn't it be ```IReadOnlyList<string?>``` ?)